css/main.css

```diff
@@ -19,12 +19,28 @@ table, th , td{
 
 .BlackBlock{
     height: 100%;
+    width: 100px;
     background: gray;
 }
 
 th, td{
     text-align: left;
 }
+
 .BoardBlock{
     height: 80px;
+    width: 100px;
+}
+
+.BlockDimension{
+	height: 100px;
```

Comment:
is there a reason why the .BoardBlock is 80px but the .BlockDimension height is 100px?

Reply:
fix spacing here too